### ISSUE TITLE
Use of multiple Markdown-Files

### ISSUE THREAD

**Issue by jwillem** [smallhadroncollider/taskell#7]
An option to split the saved output to multiple file would be great.

**Comment by smallhadroncollider** [comment]
Would you split per list?

**Comment by jwillem** [comment]
Exactly. But more like an option to choose from. As a standart in one file is perfectly fine. :)

**Comment by fullstopslash** [comment]
Plus one for this, but I'd personally only use it to archive /Done/ actions. It's nice to have a list of the things I've done, but I wouldn't want it piling up over a year, just something to look at when I'm reviewing my progress every week or so.
